class-wp-bootstrap-navwalker.php

```diff
@@ -245,7 +245,7 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 			}
 
 			/** This filter is documented in wp-includes/post-template.php */
-			$title = apply_filters( 'the_title', $item->title, $item->ID );
+			$title = apply_filters( 'the_title', esc_html( $item->title ), $item->ID );
```

Comment:
I don't think that escaping the title, at least with `esc_html()`, is a good idea. If someone wants to have HTML in their title, it will get messed up. E.g. `I <i class="fa fa-heart" aria-hidden="true"></i><span class="sr-only">love </span>U` will be rendered exactly like that and not like 'I &hearts; U'. Also, I don't think that the the_title filter hook expects an escaped string.